.codeclimate.yml

```diff
@@ -64,3 +64,7 @@ plugins:
       file: '.rubocop.yml'
     exclude_patterns:
       - db/**/*
+  reek:
```

Comment:
We should specify the same version we are using in the Gemfile. Same for `brakeman` and `rubocop` (it has 0.49 but it should be 0.65)